pkg/apis/acme/v1alpha2/types_issuer.go

```diff
@@ -64,10 +67,12 @@ type ACMEExternalAccountBinding struct {
 	// the External Account Binding keyID above.
 	// The secret key stored in the Secret **must** be un-padded, base64 URL
 	// encoded data.
+	// +kubebuilder:validation:Required
 	Key cmmeta.SecretKeySelector `json:"keySecretRef"`
 
 	// keyAlgorithm is the MAC key algorithm that the key is used for. Valid
 	// values are "HS256", "HS384" and "HS512".
+	// +kubebuilder:validation:Required
```

Comment:
This change doesn't seem to do anything - fields are already considered `required` if they are not `optional`